crates/sui-indexer/migrations/2023-03-20-041133_epoch/up.sql

```diff
@@ -25,6 +25,7 @@ FROM
   tps_data
 WHERE 
   time_diff IS NOT NULL;
+CREATE UNIQUE INDEX epoch_network_metrics_tps ON epoch_network_metrics(tps_30_days);
```

Comment:
concurrent refreshing requires unique index, otherwise it will fail due to 
```
ERROR:  cannot refresh materialized view "public.epoch_network_metrics" concurrently
HINT:  Create a unique index with no WHERE clause on one or more columns of the materialized view.
```